kittengroomer/helpers.py

```diff
@@ -236,52 +179,92 @@ def safe_copy(self, src=None, dst=None):
         if dst is None:
             dst = self.dst_path
         try:
-            dst_path, filename = os.path.split(dst)
-            if not os.path.exists(dst_path):
-                os.makedirs(dst_path)
+            # TODO: maybe don't check and just mkdir? Is there a way to do this that won't raise an Exception? Can we make shutil automatically make dirs inbetween?
```

Comment:
One thing to note, the docs for makedirs say that it can still raise an error with an existing directory if the existing dir has different permissions. This can't happen in 3.4.1 or above. I'll make the change, but maybe we should wrap it in its own try/except?